solutions/USACO jan21-uddered-bronze.mdx

```diff
@@ -0,0 +1,44 @@
+---
+id: usaco-1083
+source: USACO Bronze 2021 January
+title: Uddered but not Herd
+author: Ananth Kashyap
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_prob1_bronze_jan21.html)
+
+In this problem, we are asked to find the minimum number of times that Bessie must
+hum the entire cowphabet to have sung every letter in the given string in order.
+For every character in the string, we want to know if the next character in the
+string appears before this character in the cowphabet. If this is true, we know
+that we have to increment the number of times that Bessie must hum the cowphabet
+as Bessie cannot hum the cowphabet backwards, and thus must hum it again in order
+to reach the next character.
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(length(string))$
```

Comment:
```suggestion
**Time Complexity:** $\mathcal{O}(length(string))$
```

Reply:
? what changed?

Reply:
I told him to remove the time complexity...

Reply:
the suggestion shows up for me as removing the time complexity then adding it back in? idk maybe my github is glitchy